### DISCUSSION TITLE
Yolov4 and Yolov7 config files

### DISCUSSION THREAD

**Discussion by pfeatherstone** [ultralytics/ultralytics#5098] [Q&A]
Is there a reason why this repository doesn't provide yolov4 or yolov7 YAML config files? Is it a license issue ?

**Comment by glenn-jocher** [comment]
@pfeatherstone hello! The focus of this repository is on developing and maintaining YOLOv8, which is why you won't find config files for YOLOv4 or YOLOv7 here. Each version of YOLO is typically maintained in its own repository, and our efforts are dedicated to ensuring that YOLOv8 is the best it can be. If you're interested in YOLOv4 or YOLOv7, I would recommend checking out their respective repositories or communities for support and resources. For YOLOv8, you can find all the necessary documentation and guidance on how to use and configure the model at our [Ultralytics Docs](https://docs.ultralytics.com). 😊🚀

**Reply by pfeatherstone** [reply]
Ok cool. It's just you still provide YAML files for v3, v5 and v6. So you have everything but those two

**Reply by pderrenger** [reply]
@pfeatherstone apologies for any confusion. The inclusion of YOLOv3, YOLOv5, and YOLOv6 config files in our repository is based on the historical development and contributions within the Ultralytics ecosystem. YOLOv4 and YOLOv7 were developed by different authors and have separate repositories and communities. Our repository aims to provide a seamless experience with the models we directly support and develop, hence the focus on YOLOv8 and its predecessors within our lineage. If you're looking for YOLOv4 or YOLOv7, I'd recommend visiting their respective repositories for the most accurate and up-to-date information. 👍📚
